docs/docsite/rst/administration/instances.rst

```diff
@@ -234,6 +236,99 @@ You can remove an instance by clicking **Remove** in the Instances page, or by s
 10. To view a graphical representation of your updated topology, refer to the :ref:`ag_topology_viewer` section of this guide.
 
 
+Configuring a mesh ingress
+---------------------------
+
+If a remote execution node is setup inside a datacenter to communicate with target hosts from a k8s cluster because the k8s cluster is unable to reach the hosts via SSH, it risks exposing port information. To solve this, a hop node is placed inside of the k8s cluster to route traffic from task pods to the execution node, eliminating the risk of exposing any ports since execution node connections are outbound only.
```

Comment:
Creating a remote execution node without enabling mesh ingress you would have to allow inbound connection to the receptor listener port on the remote execution node 

In restricted networking environment (inside private network) where this connection is not allowed using mesh ingress can allow the remote execution to connect into the awx control-plane instead of having to allow connection from the awx control-plane

Reply:
`To solve this, a hop node is placed inside of the k8s cluster to route traffic from task pods to the execution node`

more of implementation detail... the ingress present as a hop node in the instance page to allow user to peer into the mesh